mmv1/templates/terraform/examples/backend_bucket_global_ilb.tf.tmpl

```diff
@@ -0,0 +1,12 @@
+resource "google_compute_backend_bucket" "{{$.PrimaryResourceId}}" {
```

Comment:
```suggestion
resource "google_project" "unarmored" {
  name = "{{$.Vars.test_project}}"
}

resource "google_compute_backend_bucket" "{{$.PrimaryResourceId}}" {
```

Reply:
Fixed creating this new project with the api needed
And run the test on top of this project.